sdk/identity/identity/src/msal/nodeFlows/msalPlugins.ts

```diff
@@ -194,6 +194,18 @@ function generatePluginConfiguration(options: MsalClientOptions): PluginConfigur
           ].join(" "),
         );
       }
+      // There is a scenario where the VSCode plugin is configured but the broker is not available (e.g., missing native dependencies).
+      // This check ensures we throw an error and not proceed with an invalid configuration.
+      if (vsCodeBrokerInfo.broker.isBrokerAvailable === false) {
```

Comment:
nit: there's a lot of validation scenarios here, consider pulling them out into a separate function to ensures things are configured correctly, just for readbility